src/pages/Stores/components/BlockStore/BlockStore.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import './styles.scss';
+
+import { Store } from '@/context/WorkspaceContext/types';
+import { useWorkspace } from '@/hooks/useWorkspace';
+
+const BlockStore = () => {
```

Comment:
We need a link somewhere that can take them to JupyterHub, as this is where they can browse the block store, maybe a link in one of the rows?